spec/models/test_track/ab_configuration_spec.rb

```diff
@@ -68,7 +68,16 @@
     it "raises a descriptive error when the split is not in the split_registry" do
       expect do
         described_class.new initialize_options.merge(split_name: :not_a_real_split)
-      end.to raise_error("unknown split: not_a_real_split")
+      end.to raise_error("unknown split: not_a_real_split.")
+    end
```

Comment:
I'm interested in the production env behavior too - possibly more Than the test env behavior. One more spec demonstrating expected behavior in production and we should be good to go!